crypto/rand/rand_lib.c

```diff
@@ -136,7 +136,6 @@ size_t rand_drbg_get_entropy(RAND_DRBG *drbg,
     size_t entropy_available = 0;
 
     RAND_POOL *pool = RAND_POOL_new(entropy, min_len, max_len);
```

Comment:
Having an empty line in 137, but none in 139 does not look right either: The unwritten coding rule "leave no empty lines between declarations, but one at the end" is probably stronger than the rule "don't leave an empty line between function call and return value check"  ;-). I will redo this commit.